src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -47,6 +54,12 @@ internal static IntellisenseQuickInfoItem BuildItem(ITrackingSpan trackingSpan,
                 quickInfoItem.Sections.Where(s => s.Kind != QuickInfoSectionKinds.Description)
                                       .Select(BuildClassifiedTextElement));
 
+            // build text for RelatedSpan
+            if (quickInfoItem.RelatedSpans.Any())
+            {
+                elements.Add(await ClassifiedSpansAndHighlightSpanFactory.BuildClassifiedTextElementForSpansAsync(quickInfoItem.RelatedSpans, snapshot, document, cancellationToken).ConfigureAwait(false));
```

Comment:
or, ideally, just return ClassifiedSpans here, and make whatever 'ClassifiedTextElements' you want here.

Reply:
fixed